### PR TITLE
add process withdraw payments feature

### PR DESCRIPTION
**Pull request by ErcilioMarques**

### CODE REVIEW

File: src/UI/pages/Payments/WithdrawPaymentsModal.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  const createPaymentData = useCallback(
+    (pendingWithdraws: IPayment[]): any[] => {
+      if (!pendingWithdraws) {
+        return [];
+      }
+
+      return pendingWithdraws.map((data: IPayment) => ({
+        order: `${data.orderNumber}`,
+        amount: <MoneyStye mode="SUCCESS">{formatCurrency(data.price)}</MoneyStye>,
+        date: dayjs(data.createdAt).format("DD/MM/YYYY HH:mm"),
```
Comment (herquiloidehele) [inline src/UI/pages/Payments/WithdrawPaymentsModal.tsx]: We already have date format inside Constants.ts

File: src/UI/pages/Payments/WithdrawPaymentsModal.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+    setModalData({ ...modalData, isOpen: false });
+    onClose(false);
+  }, [modalData]);
+
+  const initModalInfo = useCallback(() => {
+    setModalData({
+      id: "withdraw-payments-modal",
+      isOpen: false,
+      title: "Levantamento de Pagamentos",
```
Comment (herquiloidehele) [inline src/UI/pages/Payments/WithdrawPaymentsModal.tsx]: translations

**Review by herquiloidehele — CHANGES_REQUESTED** [review]

File: src/UI/pages/Payments/WithdrawPaymentsModal.tsx (hunk 1 above)
Comment (ErcilioMarques) [reply src/UI/pages/Payments/WithdrawPaymentsModal.tsx]: solved

File: src/UI/pages/Payments/WithdrawPaymentsModal.tsx (hunk 2 above)
Comment (ErcilioMarques) [reply src/UI/pages/Payments/WithdrawPaymentsModal.tsx]: solved

**Comment by herquiloidehele** [comment]
Missing the store name:

Screenshot 2024-05-05 at 20 02 44

**Review by herquiloidehele — APPROVED** [review]

**Pull request merged**